src/components/form/MultiselectForm.tsx

```diff
@@ -1,18 +1,30 @@
 import React, { PureComponent } from 'react';
 
+import _ from 'lodash';
```

Comment:
FYI hiermee importeer je lodash volledig, wat dus ook het resultaat heeft dat lodash volledig in de bundled JS terecht komt.  
Het is waarschijnlijk verstandiger om `import { sortBy } from 'lodash'` te doen, of zelfs heel lodash droppen en alleen de [sortBy](https://www.npmjs.com/package/lodash.sortby) module te installeren